css/registration.css

```diff
@@ -0,0 +1,191 @@
+.main-container {
+    display: flex;
+    justify-content: space-between;
+}
+
+.form-container {
+    padding-left: 160px;
+}
+
+.form-container h1 {
+    font-size: 60px;
+    font-weight: 500;
+    color: #E0633A;
+    max-width: 600px;
+}
+
+/* Form styling */
+
+.karma-form {
+    display: flex;
+    flex-direction: column;
```

Comment:
do you need flex box here? By default, the display block should make everything display in a column